### DISCUSSION TITLE
Conan2: which `includePath` in `c_cpp_properties.json`

### DISCUSSION THREAD

**Discussion by johhnry** [microsoft/vscode-cmake-tools#3026] [Q&A]
Hi,

I am using [Conan](https://conan.io/) version 2.0 and in VSCode I want to be able to include the libraries I installed with the `conanfile.txt`.

For version 1.x, there was a simple way as [described here](https://stackoverflow.com/questions/58077908/linking-conan-include-to-vs-code):

> Add the following line in your project's .vscode/c_cpp_properties.json file
> "includePath": ["${workspaceFolder}/**", "~/.conan/data/**"]

Now in Conan 2 it's a different folder structure so my guess is that the header files are located in `C:\Users\<user>\.conan2\p` but it contains hash names and maybe unwanted files to include. It works but it's very slow and contains a lot of files...

Do someone know a better way to integrate Conan 2 in VSCode with CMake?

Thanks!

**Comment by gcampbell-msft** [comment]
@johhnry Unfortunately, I'm not overly familiar with Conan. However, there seems to be documentation that may have better guidance: https://docs.conan.io/2/integrations/cmake.html. 

Let me know if that helps at all, and if anyone else has insight please feel free to provide info here.
